src/styles/_Form.scss

```diff
@@ -0,0 +1,39 @@
+@use 'variables' as v;
+
+.request-form {
+  height: 550px;
+  font: v.$primary-font;
+  display: flex;
+  flex-direction: column;
+  input, textarea {
+    margin: .6em 0;
+  }
+  .field-error {
+    color: red;
+  }
+  .request-comments {
+    height: 120px;
```

Comment:
Given that request form is in pixels, do you think request comments could be as a % or rem? (maybe you could have a min-height in pixels for fail-safe)

Reply:
yeah! I just changed it to min-height: 20%

the textarea element is so weird to style bc the user can change the width/height